angular.json

```diff
@@ -1,44 +1,55 @@
 {
   "$schema": "./node_modules/@angular/cli/lib/config/schema.json",
+  "cli": {
+    "analytics": false
+  },
   "version": 1,
   "newProjectRoot": "projects",
   "projects": {
-    "nereo-material": {
-      "root": "",
-      "sourceRoot": "src",
+    "cotecna-material": {
       "projectType": "application",
-      "prefix": "app",
       "schematics": {
         "@schematics/angular:component": {
           "style": "scss"
+        },
+        "@schematics/angular:application": {
+          "strict": true
         }
       },
+      "root": "",
+      "sourceRoot": "src",
+      "prefix": "app",
```

Comment:
prefix app? It should be cotecna-material or comat

Reply:
- Library Prefix set as "Comat" as Requested
- Directory and existing Prefixes Harmonization to "Comat"